Sparkle/SUBasicUpdateDriver.m

```diff
@@ -65,13 +65,14 @@ - (void)checkForUpdatesAtURL:(NSURL *)URL host:(SUHost *)aHost
     [super checkForUpdatesAtURL:URL host:aHost];
 	if ([aHost isRunningOnReadOnlyVolume])
 	{
+        NSString *hostName = [aHost name];
         if ([aHost isRunningTranslocated])
         {
-            [self abortUpdateWithError:[NSError errorWithDomain:SUSparkleErrorDomain code:SURunningTranslocated userInfo:@{ NSLocalizedDescriptionKey: [NSString stringWithFormat:SULocalizedString(@"Quit %1$@, move it into your Applications folder, relaunch it from there and try again. %2$@ can’t be updated if it’s running from the location it was downloaded to.", nil), [aHost name], [aHost name]] }]];
+            [self abortUpdateWithError:[NSError errorWithDomain:SUSparkleErrorDomain code:SURunningTranslocated userInfo:@{ NSLocalizedRecoverySuggestionErrorKey: [NSString stringWithFormat:SULocalizedString(@"Quit %1$@, move it into your Applications folder, relaunch it from there and try again.", nil), hostName], NSLocalizedDescriptionKey: [NSString stringWithFormat:SULocalizedString(@"%1$@ can’t be updated if it’s running from the location it was downloaded to.", nil), hostName], }]];
         }
         else
         {
-            [self abortUpdateWithError:[NSError errorWithDomain:SUSparkleErrorDomain code:SURunningFromDiskImageError userInfo:@{ NSLocalizedDescriptionKey: [NSString stringWithFormat:SULocalizedString(@"%1$@ can't be updated, because it was opened from a read-only or a temporary location. Use Finder to copy %1$@ to the Applications folder, relaunch it from there, and try again.", nil), [aHost name]] }]];
```

Comment:
This change is backwards. "Use Finder to copy" is the new, intended version. Finder very specifically is required here, because e.g. copying from command line or a 3rd party tool like PathFinder doesn't achieve the same effect, and we've been getting bug reports about this.

Reply:
Ah, it makes sense that the translations would lag behind the localizable string in English. I've changed the key back to the newer value.